deep-learning/_papers/2017-02-28-you-only-look-once.md

```diff
@@ -0,0 +1,52 @@
+
+
+```
+---
+layout: review
+title:  You Only Look Once: Unified, Real-Time Object Detection (YOLO)
+author: Frédéric Branchaud-Charron
+link:   https://arxiv.org/abs/1506.02640
+tags:   convolutional-networks deep-learning object-detection bounding-boxes
+---
+```
+
+YOLO v1
+
+## Summary
+
+**YOLO** is an unified solution to the object detection problem. Other approches like **R-CNN** are using a two step process *(localization and classification)* where **YOLO** is a one step process. Also, **Fast YOLO** (or tiny-yolo) is achieving of 155 fps where **Faster R-CNN** is doing 18 fps.
+
+On the Pascal VOC-2012 dataset, **YOLO** is achieving 63.4 mAP where **Faster R-CNN** is doing 73.2. In this case, **Faster R-CNN** is not real-time (7 fps) while **YOLO** is doing around 60 fps.
+
+It's important to know that **YOLO** only support a **single** aspect ratio, this is fixed in **YOLOV2** and this is the main drawback if we compare **YOLOV1** to Single shot detector (**SSD**)
```

Comment:
support a --> support**s** a

Also, I don't understand the sentence. Is the aspect ratio fixed in both versions of YOLO ?